### PR TITLE
Fix #85 Add bash script auto completion for oc, oadm and openshift

### LINKED ISSUE #85: openshift (oc) bash auto completion is missing
We should support oc auto completion which is currently not working.

``` bash
$ oc des<TAB>
$ oc describe
```

Openshift has such scripts: https://github.com/openshift/origin/tree/master/contrib/completions/bash

### PR DESCRIPTION
**Pull request by praveenkumar**
I spent enough time to find out a way to reference those file directly from 'origin' repo and tried submodule and subtree but both offer repository level referencing not file level so I just dump it from 1.1.3 label (which we are having currently).

If someone have better suggestion then I would love to hear for referencing.

@hferentschik @tnozicka please review.

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@praveenkumar awesome job! thanks

**Comment by tnozicka** [comment]
@praveenkumar Sorry for the delay. I also don't know of any way how to reference single file in git. Do we have any documentation about steps that should be done when moving ADB to newer version of OpenShift? I am afraid we might forget to update the completion files in the future.
And great job!

File: adb-utils.spec [hunk 1]
```diff
@@ -27,6 +27,8 @@ specific service and directly including it to kickstart file.
 
 %{__install} -pm644 services/openshift/openshift.service \
 %{buildroot}%{_unitdir}/openshift.service
+%{__install} -d -m 755 %{buildroot}%{_sysconfdir}/bash_completion.d/
+%{__install} -pm644 bash_completion/* %{buildroot}%{_sysconfdir}/bash_completion.d/
```
Comment (tnozicka) [inline adb-utils.spec:31]: s/-pm644/-p -m 644/ - to be consistent with the line above (30)? But it is also on line 28 which has already been there before.

Comment (praveenkumar) [inline adb-utils.spec:31]: line 30 is creating a directory and line  31 for permission of the file.

**Comment by praveenkumar** [comment]
>  Do we have any documentation about steps that should be done when moving ADB to newer version of OpenShift? I am afraid we might forget to update the completion files in the future.

@tnozicka Feel same, but I didn't know how we can keep it updated with latest release of openshift (may be every time just before release we should check if something changed upstream and bring changes to this repo).

**Comment by LalatenduMohanty** [comment]
@praveenkumar  as @tnozicka mentioned , I think we should add documentation how we should update OpenShift in ADB.

File: adb-utils.spec (hunk 1 above)
Comment (tnozicka) [inline adb-utils.spec:31]: @praveenkumar  My point was about style consistency of using short options. If we use '-p -m 775' on one line we should not use '-pm755' (group it together) on the other.

**Comment by praveenkumar** [comment]
>  I think we should add documentation how we should update OpenShift in ADB.

@LalatenduMohanty documentation add to readme or somewhere else?

**Comment by LalatenduMohanty** [comment]
@praveenkumar I am ok with anything. May be we can add steps to the doc about how we bring OpenShift templates also to adb-utils to the doc in a separate PR.

**Comment by praveenkumar** [comment]
> May be we can add steps to the doc about how we bring OpenShift templates also to adb-utils to the doc in a separate PR.

Yes I prefer it in separate PR for doc.

**Comment by LalatenduMohanty** [comment]
LGTM. Created a separate issue for documentation https://github.com/projectatomic/adb-utils/issues/114

**Pull request merged**